gretel/create_synthetic_data_from_csv_or_df/manifest.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Create synthetic data from your own CSV or DataFrame",
-    "description": "Create a synthetic dataset from an existing CSV or DataFrame with Gretel's premium SDKs.",
+    "name": "Create synthetic data from a CSV or DataFrame",
+    "description": "Use Gretel's SDK to create a synthetic dataset from a CSV or DataFrame.",
```

Comment:
"Gretel's SDK" since we don't currently have a single SDK this could be confusing.

Reply:
Good call- I updated the tex to "Use Gretel's SDKs..."